pkgs/by-name/lo/lovely-injector/package.nix

```
@@ -0,0 +1,47 @@
+{
+  pkgs,
+  fetchFromGitHub,
+  rustPlatform,
+  lib,
+}:
+let
+  version = "0.5.0-unstable";
```

```suggestion
  version = "0.5.0-unstable";
```

we need `-yyyy-mm-dd` format after the `unstable` bit, corresponding to the date of the commit